ui/app/src/modules/Content/Publish/PublishDialog.tsx

```diff
@@ -291,6 +291,15 @@ function PublishDialogUI(props: PublishDialogUIProps) {
     apiState,
     classes
   } = props;
+
+  const buttonRef = useRef<HTMLButtonElement>();
+
+  useEffect(() => {
+    if (submitDisabled === false) {
+      buttonRef.current.focus();
```

Comment:
On second thought, this could be harmful for accessibility and usability. Let’s revert.